src/DummyTest.java

```diff
@@ -1,9 +1,101 @@
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.BeforeEach;
+
+import org.junit.jupiter.params.provider.NullSource;
+
+
 
 import static org.junit.jupiter.api.Assertions.*;
 
 // FIXME: remove this class
 public class DummyTest {
+    static Entity entity1;
+    static Entity entity2;
+    
+   
+
+    @Test
+    @DisplayName("Test in Entity")
+    void testInConstructor() {
+        Entity entity = new Entity(100, 50, 40, 2, "Bob", "Bob story");
```

Comment:
Facebook chatben írtam, hogy a tesztek legyenek BDD formátumban:
```java
// Arrange
Entity entity;

// Act
entity = new Entity(...);

// Assert
assertThis(...);
assertThat(...);
```
Ugyan az `Arrange` itt triviális, jó ha megvan, hogy rögzüljön a ritmus. Akár még így is mehet:
```java
// Arrange

// Act
Entity entity = new Entity(...);
```
Így ha a jövőben hozzá kell nyúlni, akkor szebb lesz a diff és amúgyis adott a helye a dolgoknak.